twake/backend/node/README.md

```diff
@@ -1 +1,53 @@
 # Twake backend
+
+## Developer guide
+
+### ORM
+
+We use [TypeORM](https://typeorm.io/) as ORM for all the services of the Twake backend. There are several guidelines to follow in order to manage entities and listeners as described below.
+
+#### Entities
+
+1. Entities have to be defined in folders named `entities`. This comes from the typeorm API which does not allow to register new entities at runtime but allows to define `glob` expression to lookup entities in folders.
+2. Entities must follow the typeorm declaration as defined in the [entity documentation](https://typeorm.io/#/entities).
+
+```js
+import { Entity, Column, ObjectID, ObjectIdColumn } from "typeorm";
+
+@Entity({ name: "channels" })
+export class Channel {
+  @ObjectIdColumn()
+  id: ObjectID;
+
```

Comment:
Can you put an example of usage with something more "Cassandra" ?
Usually we have something like:
"scylladb_keys": {{"workspace_id":"ASC", "parent_id":"ASC", "id":"DESC"}, {"id": "DESC"}} for a primary key composed of ((workspace_id, parent_id), id) and a secondary index with only (id). We will not use secondary indexes anymore, but we will always try to use composed primary keys.
How can we describe this in TypeORM ?

Reply:
Ok you did it while implementing channels entity, so just update the README.

Reply:
The README is here to show basic samples; we will have to add more when implementation with cassandra is ready